src/views/admin/PersonManagementView.vue

```diff
@@ -133,11 +133,25 @@
           administriertVon: selectedOrganisation.value,
           searchString: searchInputKlassen.value,
         });
+        // Dropdown wasn't updated. Ideally it should be automatically updated once the selectedOrganisation holds a value.
+        klassenOptions.value = organisationStore.klassen
+          .map((org: Organisation) => ({
+            value: org.id,
+            title: org.name,
+          }))
+          .sort((a: TranslatedObject, b: TranslatedObject) => a.title.localeCompare(b.title));
```

Comment:
Die sollten eigentlich schon sortiert vom BE kommen, aber doppelt hält besser ;-)